flang/test/Transforms/DoConcurrent/basic_host.f90

```diff
@@ -0,0 +1,53 @@
+! Mark as xfail for now until we upstream the relevant part. This is just for
+! demo purposes at this point. Upstreaming this is the next step.
+! XFAIL: *
+
+! Tests mapping of a basic `do concurrent` loop to `!$omp parallel do`.
+
+! RUN: %flang_fc1 -emit-hlfir -fopenmp -fdo-concurrent-to-openmp=host %s -o - \
+! RUN:   | FileCheck %s
+! RUN: bbc -emit-hlfir -fopenmp -fdo-concurrent-to-openmp=host %s -o - \
+! RUN:   | FileCheck %s
```

Comment:
Transforms should ideally be tested using the `fir-opt` tool.

Reply:
I will add some MLIR to MLIR tests using `fir-opt` in later PRs where I will use `fir-opt`.